android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/PatientExtension.kt

```diff
@@ -211,3 +213,14 @@ fun Patient.extractOfficialIdentifier(): String? =
   if (this.hasIdentifier())
     this.identifier.firstOrNull { it.use == Identifier.IdentifierUse.OFFICIAL }?.value
   else null
+
+fun Patient.extractTypeViaDTreeMeta(): PatientType {
+  return try {
+    val tagList = this.meta.tag.filter { it.system.equals("https://d-tree.org", true) }
```

Comment:
I think we should not hard code this `"https://d-tree.org"`

Reply:
Can we obtain it from application configuration? logged in user attribute or something?

Reply:
fixed